### DISCUSSION TITLE
Dynammic Lookup Option Picker

### DISCUSSION THREAD

**Discussion by Townieeee** [Budibase/budibase#13989] [Help] [answered]
I am experimenting with a Timesheet collection form with many rows which has Date, Hours and Project.
For the Project capture, I would like to have an Option Picker for the Project field. I want to select an option from my table (tblProject) which is just a text column of many projects that changes over time.
Can anyone please point me to a sample where I am sure someone may have done this before. Many Thx.

**Comment by mikesealey** [comment, marked as answer]
Hey @Townieeee 

I've followed along to your brief, here's what I've come up with so far.

I've got two tables,`projects` (which has columns `project name` and a relationship column to timesheets) and `timesheets` (which has a `date` column, an `hours` column, and the relationship to `projects`.

In the `projects` table, I set the `project name` column to be the display column, meaning that the value for that row is what displays in the `timesheets` table.

In the design tab, I added in a `Form Block` component, and set the source to be the Timesheets table, which then creates a dropdown list for the project.  

![image](https://github.com/Budibase/budibase/assets/39269380/bf0664a8-7243-4852-81ee-4220b9e76049)

**Comment by Townieeee** [comment]
thx Mike. I'll add this in case someone has same issue. 
Make sure you follow Mike's suggestion here - as works. Do not try doing what I had. 
I had already ejected the form block as wanted to get rid of a few extra fields I had, but added the Project Field as an OptionPicker type. (which it is - but not in this case). Make sure you leave the field Project as a Text field which will automatically show as a dropdown box due to the relationship work you did on it before this point.
